pkg/util/maven/maven_command.go

```diff
@@ -74,6 +76,13 @@ func (c *Command) Do(ctx context.Context) error {
 		args = append(args, "--settings", settingsPath)
 	}
 
+	if !util.StringContainsPrefix(c.context.AdditionalArguments, "-Dmaven.artifact.threads") {
+		args = append(args, "-Dmaven.artifact.threads="+strconv.Itoa(runtime.GOMAXPROCS(0)))
+	}
+	if !util.StringContainsPrefix(c.context.AdditionalArguments, "-T") {
```

Comment:
I think that'd be more robust to compare for equality here, as `-T` is too broad, and users are generally using the `-T N` form anyway.